src/strings/PowerFxResources.en-US.resx

```diff
@@ -4928,6 +4928,14 @@
     <value>The stated function return type does not match the return type of the function body.</value>
     <comment>This error message shows up when expected return type does not match with actual return type</comment>
   </data>
+  <data name="ErrUDF_MissingReturnType" xml:space="preserve">
+    <value>Missing function return type, for example the ":Number" in "f( x:Text ):Number = Value( x );"</value>
```

Comment:
>Text [](http://example.com/codeflow?start=76&length=4)

`Text`, `Number` and `Value` should not be translated in this example, so we should tag them as `{Locked=...} in the comments.